ts/features/walletV3/transaction/components/WalletTransactionInfoSection.tsx

```diff
@@ -0,0 +1,127 @@
+
+type WalletTransactionInfoSectionProps = {
+  transaction?: Transaction | null;
+  psp?: Psp;
+  loading?: boolean;
+};
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: IOColors["grey-50"]
+  },
+  contentCard: {
+    borderRadius: IORadiusScale["1"],
+    marginVertical: IOVisualCostants.appMarginDefault
+  }
+});
+
+/**
+ * Component that shows a success message after the wallet onboarding process is completed
+ * TODO: Define the desired design of this component
+ */
+const WalletTransactionInfoSection = ({
+  transaction,
+  psp,
+  loading
+}: WalletTransactionInfoSectionProps) => (
+  <>
+    <TransactionReceiptDivider
+      height={24}
+      width={"100%"}
+      preserveAspectRatio="xMin slice" // Add this property to fit the width to the parent
+    />
+    <View
+      style={[
+        IOStyles.horizontalContentPadding,
+        styles.container,
+        IOStyles.flex
+      ]}
+    >
+      <View
+        style={[
+          styles.contentCard,
+          IOStyles.horizontalContentPadding,
+          IOStyles.bgWhite
+        ]}
+      >
+        <ListItemHeader
+          label={I18n.t("transaction.details.info.title")}
+          accessibilityLabel={I18n.t("transaction.details.info.title")}
+        />
+        {loading && (
+          <>
+            <SkeletonItem />
+            <Divider />
+            <SkeletonItem />
+            <Divider />
+            <SkeletonItem />
+          </>
+        )}
+        {!loading && transaction && (
+          <>
+            {psp?.businessName && (
+              <>
+                <ListItemInfo
+                  accessibilityLabel={I18n.t(
+                    "transaction.details.info.pspName"
+                  )}
+                  label={I18n.t("transaction.details.info.pspName")}
+                  value={psp?.businessName}
+                />
+                <Divider />
+              </>
+            )}
+            <ListItemInfo
+              accessibilityLabel={I18n.t(
+                "transaction.details.info.dateAndHour"
+              )}
+              label={I18n.t("transaction.details.info.dateAndHour")}
+              value={format(transaction.created, "DD MMMM YYYY, HH:mm:ss")}
+            />
+            <Divider />
+            <ListItemInfoCopy
+              onPress={() =>
+                clipboardSetStringWithFeedback(transaction.id.toString())
+              }
+              accessibilityLabel={I18n.t(
+                "transaction.details.info.transactionId"
+              )}
+              label={I18n.t("transaction.details.info.transactionId")}
+              value={transaction.id.toString()}
+            />
+          </>
+        )}
+      </View>
+    </View>
+  </>
+);
+
+const SkeletonItem = () => (
+  <View style={[IOStyles.flex, { paddingVertical: 12 }]}>
+    <Placeholder.Box height={16} width="80%" radius={4} />
+    <VSpacer size={8} />
+    <Placeholder.Box height={16} width="25%" radius={4} />
```

Comment:
are we sure we want to use percentage width rather than fixed-width?

Reply:
> are we sure we want to use percentage width rather than fixed-width?

Just my opinion: it's better to use percentage widths instead of fixed width values, because at least we can be sure that the placeholder will render correctly in all screen sizes (we can't predict the different screen resolutions).

Reply:
@dmnplb aren't our sizing variables based on screen size? (dp-style)